addons/medical_treatment/functions/fnc_scanMedicalItems.sqf

```diff
@@ -0,0 +1,24 @@
+#include "script_component.hpp"
+/*
+ * Author: Salluci
+ * Caches all item classnames used in ACE_Medical_Treatment_Actions
+ *
+ * Arguments:
+ * None
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * call ace_medical_treatment_fnc_scanMedicalItems
+ *
+ * Public: No
+ */
+
+private _list = ["ACE_suture"]; // ugly workaround for sutures
```

Comment:
There is surely going to be modded items that fall in this same category, perhaps we need an additional way to mark items as needed to go in this category?

Reply:
Config property good enough?

Reply:
I would say so yeah